src/views/Languages/index.jsx

```diff
@@ -0,0 +1,269 @@
+        </div>
+        <Divider light />
+        <Sidebar activeItem={language} onItemClick={this.handleDrawerToggle} />
+      </Fragment>
+    );
+    const bugs =
+      (bugzillaData &&
+        bugzillaData.bugs &&
+        uniqBy(
+          bugzillaData.bugs.edges.map(edge => edge.node).map(bug => ({
+            assignee: bug.status === 'ASSIGNED' ? bug.assignedTo.name : '-',
+            project: bug.component,
+            tags: [
+              ...(bug.keywords || []),
+              ...extractWhiteboardTags(bug.whiteboard),
+            ],
+            summary: bug.summary,
+            lastUpdated: bug.lastChanged,
+            id: bug.id,
+            url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
+          })),
+          'summary'
+        )) ||
+      [];
+
+    return (
+      <div className={classes.root}>
+        <AppBar position="fixed" className={classes.header}>
+          <Grid
+            container
+            className={classes.title}
+            alignItems="center"
+            spacing={8}>
+            <Hidden mdUp>
+              <Grid item>
+                <IconButton
+                  className={classes.button}
+                  size="large"
+                  onClick={this.handleDrawerToggle}>
+                  <MenuIcon />
+                </IconButton>
+              </Grid>
+            </Hidden>
+            <Grid item>
+              <Typography
+                align="center"
+                variant="display1"
+                noWrap
+                className={classes.link}
+                component={Link}
+                to="/">
+                Codetribute
+              </Typography>
+            </Grid>
+          </Grid>
+        </AppBar>
+        <Hidden mdUp>
+          <Drawer
+            variant="temporary"
+            anchor="left"
+            open={drawerOpen}
+            onClose={this.handleDrawerToggle}
+            classes={{
+              paper: classes.drawerPaper,
+            }}
+            ModalProps={{
+              keepMounted: true,
+            }}>
+            {drawer}
+          </Drawer>
+        </Hidden>
+        <Hidden smDown implementation="css">
+          <Drawer
+            variant="permanent"
+            open
+            PaperProps={{
+              elevation: 2,
+            }}
+            classes={{
+              paper: classes.drawerPaper,
+            }}>
+            {drawer}
+          </Drawer>
+        </Hidden>
+        <div className={classes.container}>
+          {error && <ErrorPanel error={error} />}
+          {bugzillaData &&
+            bugzillaData.error && <ErrorPanel error={bugzillaData.error} />}
+          {bugzillaData &&
+            bugzillaData.loading && <Spinner className={classes.spinner} />}
+          {(!bugzillaData || !bugzillaData.loading) && (
+            <TasksTable items={bugs} onBugInfoClick={this.handleBugInfoClick} />
+          )}
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
The only difference between this view and the project view is the GraphQL query right? To ensure we don't have a lot of duplication, I suggest we use  the same file for `/languages` and `/project`. You can then construct the query based on the URL location.

Reply:
i think another different is that we dont have the Sidebar for the `/project` and project information in the `/languages`. But yeah except to the query variables, most of them are similar

Reply:
and the css is different too because of the sidebar, wdyt? @helfi92

Reply:
Ah, in that case we can probably keep the current setup.

Reply:
Ok then 👍